src/components/shelves/shelves.scss

```diff
@@ -1,5 +1,18 @@
 .shelf-pane {
-  margin-bottom: 20px;
+  margin-bottom: 12px;
+}
+
+.shelf-encoding-pane, .shelf-filter-pane {
+  margin-bottom: 18px;
+}
+
+.shelf-encoding-pane {
+  flex-basis: 60%;
```

Comment:
I don't think this is ideal, but from testing in my machine.  This is OK for now.